pkg/thanos/statefulset.go

```diff
@@ -354,6 +354,10 @@ func makeStatefulSetSpec(tr *monitoringv1.ThanosRuler, config Config, ruleConfig
 		}
 	}
 	podLabels["app"] = thanosRulerLabel
+	podLabels["app.kubernetes.io/name"] = thanosRulerLabel
+	podLabels["app.kubernetes.io/version"] = operator.DefaultThanosVersion
```

Comment:
@simonpasquier any recommendations on what to set here (this is currently a placeholder)? unfortunately tr.spec.Version doesn't exist and I'm not sure what to set it to.

Reply:
I wouldn't add this label at all.

Reply:
Thanks, removed it.